### DISCUSSION TITLE
Curseforge api safety issues

### DISCUSSION THREAD

**Discussion by JustSimplyKyle** [gorilla-devs/ferium#165] [General]
Currently ferium's api key is provided by theRookieCoder, it's stored directly in config. Which is technically not voiding the ToS, but when someone else misuse it, the key might get removed. I wonder if it's possible to use gpgkey to encode the key. This will effectively allow people to compile ferium themselves with curseforge functionality without having to own a api key themselves, or have to make an api key public

**Comment by JustSimplyKyle** [comment]
Although you could just look back in commit and... 
Anyway, I think more safety would be better

**Comment by theRookieCoder** [comment]
CurseForge (currently at least) recommends that you leave the api key in source. They said it's fine because the api key is used for tracking, and not authentication. Regarding misuse, imo it's not my fault and I'm pretty sure CF deals with misusers by blocking their IPs.
